src/App.test.jsx

```diff
@@ -20,12 +21,20 @@ const user = {
 }
 
 // 🚨 Create your server
-
+const server = setupServer(
+  rest.get('https://uzgiamkrbapxufnwdrja.supabase.co/rest/v1/users', (req, res, ctx) => {
+    const select = req.url.searchParams.get('select')
+    if (select === '*') {
+      return res(ctx.json([user]))
+    }
+    return res(ctx.status(500))
```

Comment:
We could also send an error back here. https://mswjs.io/docs/recipes/request-assertions